PULL_REQUEST_LIFECYCLE.md

```diff
@@ -0,0 +1,28 @@
+# Pull request lifecycle
+
+* PR Template is necessary
+* PR needs to link to the issue #
+    * Issue needs to be validated (at least 1 person from the “core team” of the project)
+    * Issue needs to be assigned to the person who submitted the PR 
+
```

Comment:
```suggestion
    * Issue needs to be assigned to the person who submitted the PR 
    * **Exception:** this may not be needed if the PR is about small typo fixes
```